build.sbt

```diff
@@ -244,3 +244,15 @@ lazy val zio = (projectMatrix in file("zio"))
     settings = commonNativeSettings
   )
   .dependsOn(core)
+
+lazy val vertx = (projectMatrix in file("vertx"))
```

Comment:
thanks! also needs to be added to `allProjectRefs`

Reply:
done